integrations/ttscache-s3/src/S3TtsCache.ts

```diff
@@ -0,0 +1,153 @@
+import {
+  S3Client,
+  PutObjectCommand,
+  PutObjectCommandInput,
+  HeadObjectCommand,
+  HeadObjectCommandInput,
+  GetObjectCommand,
+  GetObjectCommandInput,
+  GetObjectCommandOutput,
+} from '@aws-sdk/client-s3';
+
+import type { Credentials } from '@aws-sdk/types';
+
+import { DeepPartial } from '@jovotech/common';
+
+import { Readable } from 'stream';
+
+import { urlJoin } from 'url-join-ts';
+
+import { TtsCachePlugin, TtsCachePluginConfig, TtsData, AudioUtilities } from '@jovotech/framework';
+
+export interface S3TtsCacheConfig extends TtsCachePluginConfig {
+  credentials: Credentials;
```

Comment:
Is it also required in a Lambda runtime?

Reply:
A Lambda runtime is not required, that is why the credential values. But is there a way to check so that if you are in a Lambda you can use those permissions? Any thoughts on how to do this? Should Credentials in config be optional?

Reply:
I was unclear, sorry. We don't need Lambda, but it should work on Lambda without adding the credentials. (like for DynamoDB)

As far as I know, all of the AWS SDKs look for the environment variables. 
Optional credentials object could be sufficient.

Reply:
It could make more sense to remove `credentials` and add `libraryConfig` like in the DynamoDB plugin. This would make it more flexible with all the config properties.
![image](https://user-images.githubusercontent.com/14978292/183687249-62a4a11e-567f-4e49-b180-db525ed77a50.png)